src/http/server.ts

```diff
@@ -744,11 +776,13 @@ export class Server {
    *
    *     Returns `undefined` if a `Drash.Http.Resource` object can't be matched.
    */
-  protected getResourceClass(request: any): Drash.Http.Resource | undefined {
+  protected getResourceClass(
+    request: Drash.Http.Request,
+  ): Drash.Interfaces.Resource | undefined {
```

Comment:
Has the doc block been updated to reflect this?

Reply:
3565ceb